src/rest/v1/openapi.yml

```diff
@@ -127,6 +129,39 @@ components:
           type: string
       required:
         - shiftUuid
+    RateWorkerInput:
+      type: object
+      properties:
+        shiftUuid:
+          type: string
+        workerUuid:
+          type: string
+        rating:
+          type: integer
```

Comment:
I'm unsure if there's a type `integer`, this would probably be a type: number with `integer` being part of the format.